hail_search/hail_search_query.py

```diff
@@ -618,6 +625,27 @@ class VariantHailTableQuery(BaseHailTableQuery):
         },
     }
 
+    @staticmethod
+    def _selected_main_transcript_expr(ht):
+        has_gene_transcripts = 'gene_transcripts' in ht.row
```

Comment:
The `gene_transcripts` is always annotated with either null or non-null values. So, is the `'gene_transcripts' in ht.row`  always `True`?

Reply:
its not always annotated, the annotation is in a method thats only conditionally called